src/sentry/static/sentry/app/views/admin/adminQuotas.tsx

```diff
@@ -4,7 +4,18 @@ import AsyncView from 'app/views/asyncView';
 import {TextField} from 'app/components/forms';
 import InternalStatChart from 'app/components/internalStatChart';
 
-export default class AdminQuotas extends AsyncView {
+type Config = {
+  backend: string;
+  options: Record<string, string>;
+};
+
+type State = AsyncView['state'] & {
+  since: number;
+  resolution: string;
+  config: Config;
```

Comment:
```suggestion
  since: number | null;
  resolution: string | null;
  config: Config | null;
```

Reply:
We set defaults on these right in the `getDefaultState` right, so these should be set no? (asside from `config`)

Reply:
yes but the AsyncComponent set it to null in case of error... I liked what the Mark said in some other PR and I'm pasting here :) 

> Having the `null` type helps reflect what can actually happen, as `AsyncComponent` sets all the state values to null when requests are made.

Reply:
> AsyncComponent sets it to null in case of error

It will just do that to `config`, not these other state attributes. And if it does error and set it to null, won't it not call `renderBody` then :)

Reply:
ok that is true, so I would add the null at least in the config :)